lib/csp.js

```diff
@@ -31,4 +33,29 @@ module.exports = function (options) {
         res.header(name, value);
         next();
     };
-};
\ No newline at end of file
+};
+
+var createPolicyString = module.exports.createPolicyString = function (policy) {
+    var entries;
+
+    if (thing.isString(policy)) {
+        return policy;
+    }
+
+    if (thing.isArray(policy)) {
+        return policy.map(createPolicyString).join('; ');
+    }
+
+    if (thing.isObject(policy)) {
+        entries = Object.keys(policy).map(function (directive) {
+            if (policy[directive] === 0 || policy[directive]) {
```

Comment:
Sure. But in this case, I also don't want other false values like `''` or `false`. I do, however, want `0` in case that value becomes meaningful in future csp directives.